.github/dependabot.yml

```diff
@@ -1,6 +1,14 @@
 version: 2
 updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "web-features"
   - package-ecosystem: "npm"
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: "web-features"
```

Comment:
@foolip should we also add these other dependencies into a group? That way dependabot only opens 1 PR for all of them. Or do you prefer separate PRs?